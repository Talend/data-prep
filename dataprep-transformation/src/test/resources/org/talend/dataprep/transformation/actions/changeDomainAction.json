{
  "actions": [
    {
      "action": "domain_change",
      "parameters": {
        "column_id": "state",
        "new_domain_id": "new_fake_domain",
        "scope": "column"
      }
    }
  ]
}