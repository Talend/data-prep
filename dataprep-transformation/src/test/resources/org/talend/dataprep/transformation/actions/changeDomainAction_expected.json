{
  "records": [
    {
      "state": "NC",
      "city":"Santa Fe",
      "tdpId": 1
    },
    {
      "state": "NY",
      "city":"Santa Fe",
      "tdpId": 2
    },
    {
      "state": "AZ",
      "city":"Santa Fe",
      "tdpId": 3
    },
    {
      "state": "CA",
      "city":"Santa Fe",
      "tdpId": 4
    }
  ],
  "columns": [
    {
      "id": "state",
      "name": "state",
      "type": "string",
      "domain": "new_fake_domain",
      "domainForced": true,
      "typeForced": false
    },
    {
      "id": "city",
      "name": "city",
      "type": "string",
      "domain": "CITY",
      "domainForced": false,
      "typeForced": false
    }

  ]
}